objects/Player/Collision_EnemyBullet.gml

```diff
@@ -2,6 +2,7 @@
 // You can write your code in this editor
 
 //If the player is ina damageable state, damage them
+audio_play_sound(sound_enemybullet_ouch, 2, 0);
```

Comment:
We should put this in the setstate function so that this only plays once on entering the respawning state.